docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/noncompliant01.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Non-compliant Code Example """
+import datetime, locale
+ 
+dt = datetime.datetime(2022, 3, 9, 12, 55, 35, 000000)
+ 
+def get_date(date):
```

Comment:
Getting the following on Flake:
expected 2 blank lines, found 1Flake8(E302)
Missing function or method docstringPylint[C0116:missing-function-docstring](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/missing-function-docstring.html)